simd/loongson/jccolext-mmi.c

```diff
@@ -184,9 +185,15 @@ void jsimd_rgb_ycc_convert_mmi(JDIMENSION image_width, JSAMPARRAY input_buf,
               "$14", "memory"
            );
       } else {
-        mmA = _mm_load_si64((__m64 *)&inptr[0]);
-        mmG = _mm_load_si64((__m64 *)&inptr[8]);
-        mmF = _mm_load_si64((__m64 *)&inptr[16]);
+        if (!(((long)inptr) & 7)) {
+          mmA = _mm_load_si64((__m64 *)&inptr[0]);
+          mmG = _mm_load_si64((__m64 *)&inptr[8]);
+          mmF = _mm_load_si64((__m64 *)&inptr[16]);
+        } else {
+          mmA = _mm_loadgs_si64((__m64 *)&inptr[0]);
+          mmG = _mm_loadgs_si64((__m64 *)&inptr[8]);
+          mmF = _mm_loadgs_si64((__m64 *)&inptr[16]);
+        }
```

Comment:
@zhanglixia-hf Can you explain the purpose of this code?  It does not seem to improve performance at all.

Reply:
OK, thanks.  I have reproduced the issue, and since it affects the current stable (2.0.x) code, I will check the alignment fix into the master branch so that it will be available in the upcoming 2.0.2 release.